dt-backend/src/model/user/user.controller.ts

```diff
@@ -0,0 +1,34 @@
+import {Controller, Get, Post, Delete, Patch, Body, Param} from "@nestjs/common";
+import { User } from '../types/user.type'
+import { UserService} from "./user.service";
+
+@Controller('user')
+export class UserController {
+
+    constructor(private readonly userService: UserService) {}
+
+    @Post('createuser')
+    async createUser (@Body() userData: {username: string; mail: string; password: string }): Promise<User> {
```

Comment:
I would extract `{username: string; mail: string; password: string }` as a type / interface! In fact - that's the DTO Objects I've been talking about in the Stream. Feel free to check them out, it's gonna make this line more compact and make everything easier to work with!

Reply:
ah now i totally get it. updated and pushing soon